src/ScillaParser.ts

```diff
@@ -96,11 +97,17 @@ export const parseScillaLibrary = async (
 };
 
 export const parseScilla = (filename: string): ParsedContract => {
-  if (!fs.existsSync(filename)) {
-    throw new Error(`${filename} doesn't exist.`);
+  const resolvedFilename = path.resolve(filename)
+  if (!fs.existsSync(resolvedFilename)) {
+    throw new Error(`${resolvedFilename} doesn't exist.`);
   }
 
-  const sexp = execSync(`scilla-fmt --sexp --human-readable ${filename}`);
+  let sexp = undefined;
+  if (process.env.USE_NATIVE_SCILLA !== undefined) {
+     sexp = execSync(`scilla-fmt --sexp --human-readable ${filename}`);
+  } else {
+     sexp = execSync(`docker run --rm -v ${resolvedFilename}:/tmp/input.scilla -i zilliqa/scilla:v0.13.3 /scilla/0/bin/scilla-fmt --sexp --human-readable /tmp/input.scilla`);
```

Comment:
Minor point, maybe move the Scilla tag into a variable, so that it can't drift between here and `ScillaChecker.ts`?

Reply:
Done :-)